srcs/utils.cpp

```diff
@@ -62,6 +62,60 @@ void ft_trim(std::string &str, const std::string cut)
 		str.clear();
 }
 
+/**
+ * ft_atoi
+ * @brief  cstdlib의 atoi의 대체함수입니다.
+ * @param  {std::string} str : 10진수 숫자의 문자열
+ * @return {int}             : 변환된 숫자
+ */
+int	ft_atoi(const std::string &str)
+{
+	unsigned long long result = 0;
```

Comment:
아래에서 result에 -1을 곱하게 될 수도 있으니 unsigned long long을 쓰면 안 될 것 같아요.
```suggestion
	long long result = 0;
```